src/rules/no-restricted-paths.js

```diff
@@ -56,78 +56,125 @@ module.exports = {
     const basePath = options.basePath || process.cwd();
     const currentFilename = context.getPhysicalFilename ? context.getPhysicalFilename() : context.getFilename();
     const matchingZones = restrictedPaths.filter((zone) => {
-      const targetPath = path.resolve(basePath, zone.target);
+      const allTargets = zone.target instanceof Array ? zone.target : [zone.target];
```

Comment:
`instanceof` should never be used.
```suggestion
      const allTargets = [].concat(zone.target);
```

Reply:
@ljharb good point 👍🏽 
fixed in https://github.com/import-js/eslint-plugin-import/pull/2466/commits/95d16861d1ec2c0f0902b57330356857ee152540